rocky/rocky/views/mixins.py

```diff
@@ -224,8 +243,22 @@ def __getitem__(self, key: Union[int, slice]) -> List[HydratedFinding]:
                 offset=offset,
                 limit=limit,
             ).items
-            ooi_references = {finding.ooi for finding in findings}
-            finding_type_references = {finding.finding_type for finding in findings}
+            if self.show_muted:
+                findings = self.get_muted_findings(findings, offset, limit)
```

Comment:
Dont you allready do a call for findings in general on line 239? This seems to just do another call to octopoes for the same?

Reply:
Yes, but that depends on the flag exclude_muted=self.exclude_muted, in that call to self.octopoes_connector.list_findings, so lets try and fix that instead of adding another query.

Reply:
Yes, the normal query is with `exclude_muted=False`
but I need also `exclude_muted=True`

To show only muted `Finding(exclude_muted=False)` - `Finding(exclude_muted=True)`

Reply:
I thought to filter the first query but it has no information about which finding is muted:
<img width="422" alt="image" src="https://github.com/minvws/nl-kat-coordination/assets/30832687/16d900f1-ea74-40dd-ab22-78075bdf04aa">